### ISSUE TITLE
Project constants

### ISSUE THREAD

**Issue by ZdenekM** [robofit/arcor2#392]
- Defined at a project level, with the same data structure as action parameters.
- Can be used as action parameters.
- Example. A user defines constant robot_speed=0.5 and sets it as a parameter for all move actions in the project. Afterward, a change of speed can be done in one place instead of changing all speed parameters separately.

```json
{
  "constants":[
    {
      "id":"robot_speed_id",
      "name":"robot_speed",
      "type": "double",
      "value":"0.5"
    }
  ]
}
```

```json
{
  "actions":[
    {
      "id":"a1",
      "name":"move_1",
      "type":"robot/move",
      "parameters":[
        {
          "id":"speed",
          "constant":"robot_speed_id"
        }
      ]
    }
  ]
}
```

```python
res = Resources()
robot = res.services("robot")
robot_speed = res.constants("robot_speed")

robot.move("action_name", robot_speed)  # other parameters omitted
```

**Comment by ZdenekM** [comment]
There is already some initial support in ARServer (AddConstant, UpdateConstant, RemoveConstant). There is no support in the Build service. In order to make constants usable as action parameters, https://github.com/robofit/arcor2/issues/309 has to be finished first.

**Comment by ZdenekM** [comment]
Probably, it would be more usable to make constants available like this (similar to action points' poses and joints):
```python
res = Resources()
const = Constants(res)

robot.move("action_name", const.robot_speed)
```

Also, this would make the main script shorter and more readable (no constants IDs visible).

**Comment by ZdenekM** [comment]
...or constants could be treated similarly to action parameters:
```python
res = Resources()
robot_speed = 0.6

robot.move("action_name", robot_speed)
```
Like this, an advanced programmer directly sees the value and can easily change it if needed.

**Comment by ZdenekM** [comment]
The `robot_speed = 0.6` variant was implemented in #424.
